Retour in the box (#242)
myparcelnl/sdk: README.md

        ->setPerson($yourShipment['name'])
        ->setPostalCode($yourShipment['postal_code'])
        ->setFullStreet($yourShipment['full_street']) 
        ->setCity($yourShipment['city'])
    );
        
    // Add each consignment to the collection created before
    $consignments
        ->addConsignment($consignment);
}
```

> ### Create return in the box

Author: Done

> This example creates a consignment and return consignment by adding them to one ```MyParcelCollection()``` and then create and download one PDF with both labels.

Reviewer: ```suggestion
This example creates a consignment and a related return consignment by adding them to one `MyParcelCollection` and then creates and downloads a single PDF file with both labels.\\\\\
```

Author: Done

```php
// Create the collection before the loop
$consignments = new MyParcelCollection();

// Loop through your shipments, adding each to the same MyParcelCollection()
foreach ($yourShipments as $yourShipment) {

    $consignment = ((ConsignmentFactory::createByCarrierId(PostNLConsignment::CARRIER_ID))
        ->setApiKey('api_key_from_MyParcel_backoffice')
        ->setCountry($yourShipment['cc'])
        ->setPerson($yourShipment['person'])
        ->setCompany($yourShipment['company'])
        ->setFullStreet($yourShipment['full_street_input'])
        ->setPostalCode($yourShipment['postal_code'])
        ->setCity($yourShipment['city'])
        ->setLabelDescription($yourShipment['label_description'])
    );
        

>     // Add the consignment to the collection and generate teh return consignment

Reviewer: ```suggestion
    // Add the consignment to the collection and generate the return consignment
```

>     // Add the consignment to the collection and generate teh return consignment

Author: Done

    $consignments
        ->addConsignment($consignment)
        ->generateReturnConsignments(

>             false,

Reviewer: Misschien hier even een commentje bij zetten, is niet duidelijk vanuit de docs waar die `false` voor is

Author: Done

            function (
                AbstractConsignment $returnConsignment,
                AbstractConsignment $parent
            ): AbstractConsignment {
                $returnConsignment->setLabelDescription(
                    'Return: ' . $parent->getLabelDescription() .

>                     ' This label is valid untill: ' . date("d-m-Y", strtotime("+ 28day"))

Reviewer: ```suggestion
                    ' This label is valid until: ' . date("d-m-Y", strtotime("+ 28day"))
```

>                     ' This label is valid untill: ' . date("d-m-Y", strtotime("+ 28day"))

Author: done
